crates/flipperzero/src/dialogs/mod.rs

```diff
@@ -28,6 +29,12 @@ pub struct DialogMessage<'a> {
     _phantom: PhantomData<&'a CStr>,
 }
 
+/// A dialog file browser options.
+pub struct DialogFileBrowserOptions<'a> {
+    data: sys::DialogsFileBrowserOptions,
+    _phantom: PhantomData<&'a c_void>,
```

Comment:
Extra small detail: I guess `()` is a more common type to be used in `PhantomReference`:

```suggestion
    _phantom: PhantomData<&'a ()>,
```